lib/feature/pokemon_details_page.dart/pokemon_details_page.dart

```diff
@@ -14,13 +17,13 @@ class PokemonDetails extends StatelessWidget {
         slivers: <Widget>[
           SliverAppBar(
             leading: BackButton(color: Colors.black),
-            backgroundColor: Colors.white, // TODO: Make dynamic later
+            backgroundColor: pokemon.types?.first.type?.name?.getPokemonColor ?? ''.getPokemonColor,
```

Comment:
for readability
`pokemon.types?.first.type?.name?.getPokemonColor ?? ''.getPokemonColor`
and do the same for the rest